docs/conf.py

```diff
@@ -72,14 +72,13 @@
     import subprocess
     subprocess.call('git fetch origin --unshallow', cwd=docs_dir, shell=True)
     subprocess.check_call('git fetch origin --tags', cwd=docs_dir, shell=True)
-    subprocess.check_call('make links', cwd=docs_dir, shell=True)
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = re.sub('^v', '', os.popen('git describe ').read().strip())
+release = re.sub('^v', '', os.popen('git describe --always').read().strip())
```

Comment:
I have pushed a `v0.0` tag on the first commit in the fasm repository so that this should now work.